cluster/prod/knative/summarizer.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: testgrid-summarizer
+  namespace: knative
+  labels:
+    component: summarizer
+    app: testgrid
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: testgrid
+      component: summarizer
+  template:
+    metadata:
+      labels:
+        component: summarizer
+        app: testgrid
+    spec:
+      serviceAccountName: summarizer
+      containers:
+      - name: summarizer
+        image: gcr.io/k8s-testgrid/summarizer:v20210201-v0.0.45-2-ga6373ea
+        args:
+        - --config=gs://knative-own-testgrid/config
+        - --confirm
+        - --wait=5m
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: beta-summarizer
+  namespace: knative
+  labels:
+    component: summarizer-beta
+    app: testgrid
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: testgrid
+      component: summarizer-beta
+  template:
+    metadata:
+      labels:
+        component: summarizer-beta
+        app: testgrid
+    spec:
+      serviceAccountName: summarizer
+      containers:
+      - name: summarizer
+        image: gcr.io/k8s-testgrid/summarizer:v20210201-v0.0.45-2-ga6373ea
+        args:
+        - --config=gs://knative-own-testgrid/config
+        - --confirm
+        - --grid-path=grid
+        - --summary-path=summary
+        - --wait=5m
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    # Uses same as updater
+    iam.gke.io/gcp-service-account: updater@k8s-testgrid.iam.gserviceaccount.com
```

Comment:
Please bind this K8S SA to a different GCP SA

`serviceAccount:k8s-testgrid.svc.id.goog[knative/summarizer]` is not authorized to act on behalf of `updater@k8s-testgrid.iam.gserviceaccount.com` and shouldn't act as this entity.


I do not want the knative testgrid updater to be able to accidentally write to the core testgrid

AKA this involves:
* Reusing some other identity (who writes to gs://knative-testgrid, for example?)
* Follow https://github.com/kubernetes/test-infra/tree/master/workload-identity to authorize `serviceAccount:k8s-testgrid.svc.id.goog[knative/summarizer]` to act as `someone@some-knative-project.iam.gserviceaccount.com`
 - Need to also authorize `...[knative/updater]` to act as this entity.

Reply:
Done:
```
Updated IAM policy for serviceAccount [testgrid-updater@knative-tests.iam.gserviceaccount.com].
bindings:
- members:
  - serviceAccount:k8s-testgrid.svc.id.goog[knative/summarizer]
  - serviceAccount:k8s-testgrid.svc.id.goog[knative/updater]
  role: roles/iam.workloadIdentityUser
etag: BwW8HwYoOdY=
version: 1
```